### DISCUSSION TITLE
Blockscout can't fetch block logs "message":"Batch of empty blocks is sanitized","metadata":{"fetcher":"empty_blocks_to_refetch"}}

### DISCUSSION THREAD

**Discussion by DGSInfra** [blockscout/blockscout#8989] [Q&A]
hi guys i was using docker-compose to manual deploy blockscout and connect to my rpc. i try curl -v and ping its can connect to rpc but blockscout seems can't fetch block from rpc keeping telling this error
"message":"Batch of empty blocks is sanitized","metadata":{"fetcher":"empty_blocks_to_refetch"}} 
can anyone help me pls?
![image](https://github.com/blockscout/blockscout/assets/141704620/20515c0f-5fd6-4aaf-9499-cbec0197fb7e)

**Comment by sl1depengwyn** [comment]
This is not an error, just an info message. Do you have any blocks in blockscout?
